src/gtk-3.20/scss/widgets/_view.scss

```diff
@@ -68,16 +68,16 @@
     }
 
     %column_header_button {
+        @extend %button;
         padding: ($spacing - 2px) ($spacing + 1px);
         border-radius: 0;
-        background-image: none;
         text-shadow: none;
-        border-style: none solid solid none;
         border-color: $borders_color;
```

Comment:
i think you'll need also to remove at least this line

Reply:
and also return removed `border-style: none solid solid none;` because now border width of header buttons became twice bigger

Reply:
there's a catch here: if we make the border this way, when the column is selected its border changes color, but only at the bottom and right, which looks weird:
![screen1](https://user-images.githubusercontent.com/20945263/36075893-2bde4120-0f55-11e8-98ad-5dfc483bb0c0.jpg)
As it is now, the selected border is complete, but there il the "double" border when not selected, which is not ideal either.

I think I have a solution: we can make the top and left border transparent, but only when it is not selected:
![screen2](https://user-images.githubusercontent.com/20945263/36075934-af9ca9f2-0f55-11e8-830b-553e9cd10033.jpg)
What do you think?